StatePrinter/Introspection/IntroSpector.cs

```diff
@@ -201,15 +203,14 @@ private bool IntrospectIEnumerable(object source, Field field)
             seenBefore.TryGetValue(source, out optionReferenceInfo);
 
             tokens.Add(new Token(TokenType.FieldnameWithTypeAndReference, field, null, optionReferenceInfo, source.GetType()));
-            tokens.Add(StartEnumeration);
+            tokens.Add(StartList);
 
-            int i = 0;
+            long i = 0;
```

Comment:
how big a graph are you planning on dumping?;) Our memory consumption will increase with this change. How about using a uint? unsigned i guess is nicer in that the index can not be negative - just like an unsigned cant be negative